packages/app/src/systems/SignUp/pages/TermsOfUse/TermsOfUse.tsx

```diff
@@ -1,7 +1,10 @@
-import { Box, Button, Flex, Stack } from '@fuel-ui/react';
+import { cssObj } from '@fuel-ui/css';
+import { Box, Button } from '@fuel-ui/react';
 import { useNavigate, useParams } from 'react-router-dom';
 
 import { Header } from '../../components';
+import { Stepper } from '../../components/Stepper';
```

Comment:
can we import from `../../components` only ?

Reply:
✅